### PR TITLE
Added userscript

### PR DESCRIPTION
**Pull request by KOLANICH**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

It's a platform. A lot of repos contain userscripts.

### CODE REVIEW

File: topics/userscript/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: '2005-03-28'
+short_description: A kind of lightweight browser extension, used to modify webpages
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension
+---
+A userscript is a kind of lightweight and less priveleged browser extension written in ECMAScript used to modify webpages. Userscripts may use some API usually referred as ```GM_API``` allowing them to bypass same-origin policy and store the data in a separate storage. The technology was initially introduced in GreaseMonkey extension for Firefox web browser and then spread to other browsers, sometimes built-in, like in Presto-based Opera, sometimes with help of extensions, like in Chromium-based browsers. The most famous catalogs of userscripts are [GreasyFork](https://greasyfork.org/) and [OpenUserJS](https://openuserjs.org/).
```
Comment (MikeMcQuaid) [inline topics/userscript/index.md]: - JavaScript rather than ECMAScript may be more accessible?
- Just use one backtick around either side of `GM_API`.

**Comment by MikeMcQuaid** [comment]
@KOLANICH Thanks for the topic! Can you fix up the failing tests: https://travis-ci.org/github/explore/builds/300436972?utm_source=github_status&utm_medium=notification

File: topics/userscript/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+github_url: https://github.com/greasemonkey/greasemonkey
+released: March 28, 2005
+short_description: A kind of lightweight browser extension, used to modify webpages.
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension, JavaScript
+---
+A userscript is a kind of lightweight and less priveleged browser extension written in JavaScript used to modify webpages. Userscripts may use some API usually referred as [`GM_API`](https://wiki.greasespot.net/Greasemonkey_Manual:API) allowing them to bypass [same-origin policy](https://en.wikipedia.org/wiki/Same-origin_policy) and store the data in a separate storage. The technology was initially introduced in [GreaseMonkey](https://github.com/greasemonkey/greasemonkey) extension for [Firefox web browser](https://www.mozilla.org/firefox/), and then spread to other browsers, sometimes built-in, like in [Presto-based Opera](https://en.wikipedia.org/wiki/Presto_\(layout_engine\)), sometimes with help of extensions, like in [Chromium](https://www.chromium.org)-based browsers. The most famous catalogs of userscripts are [GreasyFork](https://greasyfork.org/) and [OpenUserJS](https://openuserjs.org/).
```
Comment (cheshire137) [inline topics/userscript/index.md]: `priveleged` => `privileged`

Could you add a comma after "API" and "GM_API", so it reads "Userscripts may use some API, usually referred as GM_API, allowing them..."?

I think you should drop the "a" in "store the data in a separate storage".

How about adding "the" to "in GreaseMonkey extension"?

File: topics/userscript/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: userjs, greasemonkey, tampermonkey, violentmonkey
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: March 28, 2005
+short_description: A kind of lightweight browser extension, used to modify webpages.
```
Comment (cheshire137) [inline topics/userscript/index.md]: Could you rephrase this as a full sentence? Maybe "A userscript is a kind of lightweight browser extension that is used to modify webpages."

File: topics/userscript/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension, JavaScript
```
Comment (cheshire137) [inline topics/userscript/index.md]: Could you lowercase these? https://github.com/topics/webextension and https://github.com/topics/javascript are the two topics.

File: topics/userscript/index.md (hunk 2 above)
Comment (KOLANICH) [reply topics/userscript/index.md]: 🤦. Done.

File: topics/userscript/index.md (hunk 4 above)
Comment (KOLANICH) [reply topics/userscript/index.md]: done

File: topics/userscript/index.md (hunk 3 above)
Comment (KOLANICH) [reply topics/userscript/index.md]: done

**Comment by MikeMcQuaid** [comment]
Thanks again for the PR, @KOLANICH! You rock!
